tests/fv3_conf/ccpp_regional_FA_run.IN

```diff
@@ -0,0 +1,31 @@
+rsync -arv @[RTPWD]/FV3_regional_input_data/. .
+rsync -arv @[RTPWD]/@[INPUT_DIR]/model_configure .
+
+rm -rf INPUT RESTART
+mkdir  INPUT RESTART
+
+rsync -arv @[RTPWD]/@[INPUT_DIR]/INPUT/. INPUT/.
+
+if [ $WARM_START = .T. ]; then
+    cp ../fv3_ccpp_regional_control${RT_SUFFIX}/RESTART/20181015.120000.coupler.res             INPUT/coupler.res
+    cp ../fv3_ccpp_regional_control${RT_SUFFIX}/RESTART/20181015.120000.fv_core.res.nc          INPUT/fv_core.res.nc
+    cp ../fv3_ccpp_regional_control${RT_SUFFIX}/RESTART/20181015.120000.fv_core.res.tile1.nc    INPUT/fv_core.res.tile1.nc
+    cp ../fv3_ccpp_regional_control${RT_SUFFIX}/RESTART/20181015.120000.fv_srf_wnd.res.tile1.nc INPUT/fv_srf_wnd.res.tile1.nc
+    cp ../fv3_ccpp_regional_control${RT_SUFFIX}/RESTART/20181015.120000.fv_tracer.res.tile1.nc  INPUT/fv_tracer.res.tile1.nc
+    cp ../fv3_ccpp_regional_control${RT_SUFFIX}/RESTART/20181015.120000.phy_data.nc             INPUT/phy_data.nc
+    cp ../fv3_ccpp_regional_control${RT_SUFFIX}/RESTART/20181015.120000.sfc_data.nc             INPUT/sfc_data.nc
+fi
+cp    @[RTPWD]/FV3_input_data/DETAMPNEW_DATA* .
+cp    @[RTPWD]/FV3_input_data/*grb .
+cp    @[RTPWD]/FV3_input_data/*_table .
+cp    /home/Man.Zhang/diag_table_FA diag_table
```

Comment:
This is pointing to Man's personal directory. Should it point somewhere else?

Reply:
This should be staged in the gmtb area, independent of a user, until it gets moved into the operational regression test baseline maintained by EMC.